.github/workflows/publish.yml

```diff
@@ -0,0 +1,43 @@
+name: Version and Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      newversion:
+        description: 'Version'
+        required: true
+
+concurrency:
+  group: npm-bump
+  cancel-in-progress: true
+
+jobs:
+  version_and_release:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 18
+        registry-url: 'https://registry.npmjs.org'
+
+    - uses: pnpm/action-setup@v2
+      id: pnpm-install
+      with:
+        version: 7
+
+    - run: pnpm install
+
+    - run: pnpm test
```

Comment:
Should we run `lint` and `format:check` here as well?

Reply:
I thought it might block a release, that's why I intentionally left it out.

Reply:
Wouldn't `lint` still be good to run as it would catch any unused variables, inconsistent code, etc, just in case?